tests/api.c

```diff
@@ -4462,6 +4462,10 @@ static int test_wolfSSL_EVP_get_cipherbynid(void)
 #endif
 #endif /* !NO_DES3 */
 
+#if defined(HAVE_CHACHA) && defined(HAVE_POLY1305)
+    AssertNotNull(strcmp("EVP_CHACHA20_POLY13O5", wolfSSL_EVP_getcipherbynid(1018)));
```

Comment:
Should be `wolfSSL_EVP_get_cipherbynid` not `wolfSSL_EVP_getcipherbynid`. And for the API.c test use `EVP_get_cipherbynid` to exercise the compatibility macro.